packages/instrumenter/test/unit/mutators/unary-operator-mutator.spec.ts

```diff
@@ -27,4 +27,14 @@ describe(sut.name, () => {
   it('should not mutate a-a', () => {
     expectJSMutation(sut, 'a-a');
   });
+
+  it('should not mutate -b to +b', () => {
+    const unaryOperatorLevelA = { name: 'unaryOperatorA', UnaryOperator: ['+To-', 'remove~'] };
+    expectJSMutationWithLevel(sut, unaryOperatorLevelA.UnaryOperator, '+a; -b; ~c;', '-a; -b; ~c;', '+a; -b; c;');
+  });
+
+  it('should only mutate -b to +b', () => {
+    const unaryOperatorLevelB = { name: 'unaryOperatorB', UnaryOperator: ['-To+'] };
+    expectJSMutationWithLevel(sut, unaryOperatorLevelB.UnaryOperator, '+a; -b; ~c;', '+a; +b; ~c;');
+  });
```

Comment:
I think two more tests could be useful: when there are no UnaryOperator sub-operators, and when all the operators are there.